engine/client/cl_main.c

```diff
@@ -1374,6 +1374,11 @@ void CL_ClearState( void )
 	Cvar_SetValue( "scr_download", -1.0f );
 	Cvar_SetValue( "scr_loading", 0.0f );
 	host.allow_console = host.allow_console_init;
+	if( !SV_Active( ) && !CL_IsPlaybackDemo( ) && !cls.demorecording )
+	{
+		Delta_Shutdown( );
```

Comment:
As I'm playing on old servers sometimes, I've never encountered this issue. Is it gone now? I don't know.

Reply:
I doubt, it's happens only after changelevel, especially on co-op servers. And I faced with this bug not so long ago. May be it's worth to test it again now.